client/modules/IDE/components/Editor.jsx

```diff
@@ -346,25 +346,23 @@ class Editor extends React.Component {
         }
       );
       this._cm.doc.setValue(formatted);
-      return cursorOffset;
+      if (cursorOffset) {
```

Comment:
Hey @catarak ,
please also add `this._cm.focus()` , before setting the cursor.

Reply:
In my testing I found this wasn't necessary? Could be Mac only. I'll try it on Windows too.

Reply:
This is necessary if we chose to use the menu for tidying the code.

Reply:
of course, thank you!